src/config/rc-files.md

```diff
@@ -0,0 +1,61 @@
+# rc.conf, rc.local and rc.shutdown
+
+The files `/etc/rc.conf`, `/etc/rc.local` and `/etc/rc.shutdown` can be used to
+configure certain parts of your Void system.
```

Comment:
Perhaps mention that `/etc/rc.conf` is often configured by void-installer ?

Reply:
Added in https://github.com/void-linux/void-docs/commit/d010d217ab8788d2feffb6711fa2825f53fbe4c3.